docker/secretflow_entrypoint.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import subprocess
+import sys 
+import re
```

Comment:
Please check again, `import re` should be in the front of `import subsprocess`.

And [isort](https://pycqa.github.io/isort/) might be helpful.

Reply:
I apologize for my negligence. I fix the issue in newst commit.Please you review again.